layers/meta-balena-beaglebone/conf/layer.conf

```diff
@@ -5,7 +5,7 @@ BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
 
 BBFILE_COLLECTIONS += "balena-beaglebone"
 BBFILE_PATTERN_balena-beaglebone := "^${LAYERDIR}/"
-BBFILE_PRIORITY_balena-beaglebone = "1337"
+BBFILE_PRIORITY_balena-beaglebone = "1338"
```

Comment:
I don't think it's needed to change the priority here. Any reason for the increase?

Reply:
I added this as part of trying to disable the `systemd-watchdog` (which is actually set in the `meta-balena` layer, whose BBFILE_PRIORITY I was trying to override). This is no longer required, because I found the *correct* way to do it (using the `watchdog-disable.conf` file.

Reply:
Reverting this change (i.e. setting the priority back to 1337)